.golangci.yml

```diff
@@ -0,0 +1,68 @@
+# This file contains all available configuration options
+# with their default values.
+
+# options for analysis running
+run:
+
+# output configuration options
+output:
+  # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
+  format: colored-line-number
+
+linters-settings:
+  funlen:
+    lines: 120
+    statements: 120
+  dupl:
+    threshold: 200
+  gocognit:
+    min-complexity: 32
+
+linters:
+  enable:
+    - bodyclose
+    - depguard
+    - dogsled
+    - errcheck
+    - exportloopref
+    - funlen
+    - gochecknoinits
```

Comment:
Ultimately, we should add `- gochecknoglobals` here too. But, until we have no globals, it's fine to leave it out.